articles/virtual-desktop/windows-10-multisession-faq.yml

```diff
@@ -27,8 +27,8 @@ sections:
         answer: |
           Windows 10 Enterprise multi-session is a virtual edition of Windows 10 Enterprise. One of the differences is that this operating system (OS) reports the [ProductType](/windows/win32/cimwin32prov/win32-operatingsystem) as having a value of 3, the same value as Windows Server. This property keeps the OS compatible with existing RDSH management tooling, RDSH multi-session-aware applications, and mostly low-level system performance optimizations for RDSH environments. Some application installers can block installation on Windows 10 multi-session depending on whether they detect the ProductType is set to Client. If your app won't install, contact your application vendor for an updated version.
           
-      - question: Can I run Windows 10 Enterprise multi-session on-premises?
-        answer: Windows 10 Enterprise multi-session can't run in on-premises production environments because it's optimized for the Azure Virtual Desktop service for Azure. It's against the licensing agreement to run Windows 10 Enterprise multi-session outside of Azure for production purposes. Windows 10 Enterprise multi-session won't activate against on-premises Key Management Services (KMS).
+      - question: Can I run Windows 10 or Windows 11 Enterprise multi-session outside of the Azure Virtual Desktop service?
+        answer: Windows 10 or 11 Enterprise multi-session is not permitted in production environments outside of the Azure Virtual Desktop service, which is provided only by Microsoft or the Azure Virtual Desktop Approved Providers Citrix and VMWare, and it is against the licensing agreement to run Windows 10 or 11 multi-session outside of the Azure Virtual Desktop Service for production purposes. In addition, Windows 10 and 11 multisession won’t activate against on-premises Key Management Services (KMS).
```

Comment:
```suggestion
        answer: We don't allow customers to run Windows 10 or 11 Enterprise multi-session in production environments outside of the Azure Virtual Desktop service. Only Microsoft or the Azure Virtual Desktop Approved Providers, Citrix and VMWare, can provide access to the Azure Virtual Desktop service. It's against the licensing agreement to run Windows 10 or 11 multi-session outside of the Azure Virtual Desktop service for production purposes. Windows 10 and 11 multi-session also won’t activate against on-premises Key Management Services (KMS).
```

Reply:
The case for VMware is wrong here - lowercase w